### ISSUE TITLE
Release twisted mail 0.3.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11213]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1732|
|Type|task|
|Created|2006-05-21 22:34:21Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1732 1732
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1148250861000000 1148250861000000
changetime__1148253968000000 1148253968000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16904]) News and Versioning from Twisted Mail 0.3.0

Author: Release Automation, exarkun
Reviewer: exarkun, radix
Fixes [#11213](https://github.com/twisted/twisted/issues/11213)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
